index.html

```diff
@@ -24,7 +24,35 @@ <h1>Scientific Modeling out of distribution (Scientific-Mood) ML Challenge</h1>
         
         <br>
         <p1>
-            The HDR ML Challenge program is hosting its second FAIR challenge, this year presenting three scientific benchmarks for modeling out of distribution in three critical areas: Neural Forecasting, Climate Prediction using Ecological Data, and Coastal Flooding Prediction over time.
+	The HDR ML Challenge program is hosting its second FAIR
+        challenge, this year presenting three scientific benchmarks
+        for modeling out of distribution in three critical areas:
+        Neural Forecasting, Climate Prediction using Ecological Data,
+	  and Coastal Flooding Prediction over time.
+
+	Machine learning problems are often driven by the quality of
+        the training dataset. They are very effect at interpolating
+        across the dataset to find patterns and trends. In this
+        challenge, we ask Machine Learning to go out of the dataset
+        by performing out of domain exptrapolatoin to prtrical
+        critical scientific process that have not yet been well
+        studies. 
```

Comment:
```suggestion
	Machine learning problems are often driven by the quality of
        the available training datasets. Models are very effective at interpolating
        across their training datasets to find patterns and trends. In this
        challenge, we ask models to extend beyond their training
        by performing out of domain extrapolation to practical
        critical scientific process that have not yet been well
        studied. 
```
Think that was supposed to be "practical", not "particle" (as suggested by autocorrect)?